File: src/main/java/com/leijendary/spring/boot/template/api/v1/data/SampleResponse.java
package com.leijendary.spring.boot.template.api.v1.data;

import java.time.OffsetDateTime;

import com.leijendary.spring.boot.template.data.LocalizedData;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class SampleResponse extends LocalizedData<SampleTranslationResponse> {

    private String column1;
    private String column2;
    private OffsetDateTime createdAt;
    private String createdBy;
    private OffsetDateTime lastModifiedAt;
    private String lastModifiedBy;
}

File: src/main/kotlin/com/leijendary/spring/boot/template/api/v1/data/SampleResponse.kt
package com.leijendary.spring.boot.template.api.v1.data

import com.leijendary.spring.boot.template.core.data.LocalizedData
import java.math.BigDecimal
import java.time.OffsetDateTime

data class SampleResponse(
    val column1: String,
    val column2: Int,
    val amount: BigDecimal,
    val createdAt: OffsetDateTime,
    val createdBy: String,
    val lastModifiedAt: OffsetDateTime,
    val lastModifiedBy: String,
) : LocalizedData<SampleTranslationResponse>()